crates/sui-adapter-transactional-tests/tests/programmable_transaction_examples/coin_limit.move

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//# init --addresses test=0x0 --accounts A
+
+//# publish
+module test::m1 {
+    use sui::object::{Self, UID};
+    use sui::tx_context::{Self, TxContext};
+    use sui::transfer;
+    use sui::coin::{Self, Coin};
+    use sui::sui::SUI;
+
+    struct CoolMarker has key, store { id: UID }
+
+    public entry fun purchase(coin: Coin<SUI>, ctx: &mut TxContext) {
+        transfer::transfer(purchase_(coin, ctx), tx_context::sender(ctx))
+    }
+
+    public fun purchase_(coin: Coin<SUI>, ctx: &mut TxContext): CoolMarker {
+        assert!(coin::value(&coin) >= 100, 0);
+        transfer::transfer(coin, @0x70DD);
+        CoolMarker { id: object::new(ctx) }
+    }
+
+}
+
+// call an entry function
+//# programmable --sender A --inputs 100
+
+0: SplitCoin(Gas, Input(0)); // split the coin as a limit
+1: test::m1::purchase(Result(0));
+
+//# view-object 107
+
+//# view-object 108
+
+// call a non-entry function, but forget the object
+//# programmable --sender A --inputs 100
+
+0: SplitCoin(Gas, Input(0)); /* split the coin as a limit */
```

Comment:
Can we make these comments in Move?  Otherwise our editors are going to get unhappy.

Reply:
Seeing as you use the same comment syntax for the command parser, how about `//>` as the prefix for the command, which can get its own token?

Reply:
Ah! I like it, will do